src/Authenticator/ResultInterface.php

```diff
@@ -15,34 +15,29 @@
 interface ResultInterface
 {
     /**
-     * General Failure
+     * Failure due to invalid credentials being supplied.
      */
-    const FAILURE = 0;
+    const FAILURE_CREDENTIALS_INVALID = 'FAILURE_CREDENTIALS_INVALID';
```

Comment:
git/github diff doesn't detect the moving around of constants correctly. Take a look at the whole constants before/after